frontend/src/components/Teams/TeamsList/partials/CardTeam/CardBody.tsx

```diff
@@ -46,9 +46,7 @@ const CardBody = React.memo<CardBodyProps>(({ userId, team }) => {
 
     const myUser = team.users.find((user) => String(user.user?._id) === String(userId));
 
-    return (
-      team && (myUser?.role === TeamUserRoles.ADMIN || myUser?.role === TeamUserRoles.STAKEHOLDER)
-    );
+    return team && [TeamUserRoles.ADMIN, TeamUserRoles.STAKEHOLDER].includes(myUser!.role);
```

Comment:
```suggestion
    if (!myUser) {
      return false
    }
    
     return team && [TeamUserRoles.ADMIN, TeamUserRoles.STAKEHOLDER].includes(myUser.role);
```

Be careful on force unwrapping the `myUser`. You can check if the user isn't found and then return false